### DISCUSSION TITLE
Chronological sort instead of Alpha sort

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#70] [Feature requests] [answered]
I noticed that sorts are alpha, as opposed to chronological in some areas. I think it would be better for it to be a date based sort.

An example where alpha sort is used is Shows

**Comment by roxsman** [comment]
Yes, this would be a very welcome feature. Makes it much easier to find the latest Tech Talk for example.

**Comment by f1multiviewer** [comment, marked as answer]
Done!

**Comment by HanSooloo** [comment]
2010 through 2017 seasons' race lists are still in AZ order, rather than race date order.  Beginning in 2018, the races are listed in race date order.
Possible to update the older races' sort oder?
